### PR TITLE
Add example to StronglyEntanglingLayers

### PR DESCRIPTION
**Pull request by ankit27kh**
**Description of the Change:**
Add example in documentation
**Benefits:**
better documentation
**Possible Drawbacks:**
none
**Related GitHub Issues:**
#1735 #1727

### CODE REVIEW

**Comment by ankit27kh** [comment]
Hey @antalszava, this should be ready.

File: pennylane/templates/layers/strongly_entangling.py [hunk 1]
```diff
@@ -50,6 +50,57 @@ class StronglyEntanglingLayers(Operation):
...
+            def circuit(parameters):
+                qml.StronglyEntanglingLayers(weights=parameters, wires=range(4))
+                return qml.expval(qml.PauliZ(0))
+
+            shape = qml.StronglyEntanglingLayers.shape(n_layers=2, n_wires=4)
+            weights = np.random.random(size=shape)
+
+        The shape of the ``weights`` argument decides the number of layers.
+
+        The resulting circuit:
```
Comment (antalszava) [inline pennylane/templates/layers/strongly_entangling.py:73]:
```suggestion
        The resulting circuit is:
```

File: pennylane/templates/layers/strongly_entangling.py [hunk 2]
```diff
@@ -50,6 +50,57 @@ class StronglyEntanglingLayers(Operation):
...
+
+            @qml.qnode(dev)
+            def circuit(parameters):
+                qml.StronglyEntanglingLayers(weights=parameters, wires=range(4), ranges=[2, 3], imprimitive=qml.ops.CZ)
+                return qml.expval(qml.PauliZ(0))
+
+            shape = qml.StronglyEntanglingLayers.shape(n_layers=2, n_wires=4)
+            weights = np.random.random(size=shape)
+
+        The resulting circuit:
```
Comment (antalszava) [inline pennylane/templates/layers/strongly_entangling.py:96]:
```suggestion
        The resulting circuit is:
```

**Review by antalszava — APPROVED** [review]
Looks good, thank you! :slightly_smiling_face:

File: pennylane/templates/layers/strongly_entangling.py [hunk 3]
```diff
@@ -50,6 +50,71 @@ class StronglyEntanglingLayers(Operation):
...
+
+        The default two-qubit gate used is :class:`~pennylane.ops.CNOT`. This can be changed by using the ``imprimitive`` argument.
+
+        The ``ranges`` argument takes an integer sequence where each element
+        determines the range hyperparameter for each layer. This range hyperparameter
+        is the difference of the wire indices representing the two qubits the
+        ``imprimitive`` gate acts on. For example, for ``range=[2,3]`` the
+        first layer will have a range parameter of ``2`` and the second layer will
+        have a range parameter of ``3``.
+
```
Comment (antalszava) [inline pennylane/templates/layers/strongly_entangling.py:89]:
```suggestion

```

**Comment by ankit27kh** [comment]
@antalszava, thank you for your input!
About the failed codefactor, it's an unrelated and unedited file. Don't know why that is showing up here.

**Comment by antalszava** [comment]
Hi @ankit27kh, no worries :slightly_smiling_face: Sure thing, we'll be merging this. Thanks again!

**Pull request merged**